.github/workflows/matrix-bot.yml

```diff
@@ -16,3 +16,16 @@ jobs:
          access_token: ${{ secrets.MATRIX_ACCESS_TOKEN }}
          message: "New PR: [${{ github.event.pull_request.title }}](${{ github.event.pull_request.html_url }})"
          server: "matrix.org"
+
+  merged-pr:
+    if: github.event.action == 'closed' && github.event.pull_request.merged == true && github.repository == 'atsamd-rs/atsamd'
```

Comment:
Maybe we want closed but not merged PRs too?

Reply:
Doneskies